src/pages/resultsView/enrichments/EnrichmentsUtil.tsx

```diff
@@ -1173,6 +1571,85 @@ export function getGeneListOptions(
     ];
 }
 
+export function getGaBinarydataListOptions(
+    data: GenericAssayBinaryEnrichmentRow[],
+    includeAlteration?: boolean
+): { label: GaBinaryOptionLabel; entities: string[] }[] {
+    if (_.isEmpty(data)) {
+        return [
+            {
+                label: GaBinaryOptionLabel.SYNC_WITH_TABLE,
+                entities: [],
+            },
+        ];
+    }
+
+    let dataWithOptionName: (GenericAssayBinaryEnrichmentRow & {
+        optionName?: string;
+    })[] = data;
+
+    let dataSortedByAlteredPercentage = _.clone(dataWithOptionName).sort(
+        compareByAlterationPercentage
+    );
+
+    let dataSortedByAvgFrequency = _.clone(dataWithOptionName).sort(function(
+        kv1,
+        kv2
+    ) {
+        const t1 =
+            _.sumBy(_.values(kv1.groupsSet), count => count.alteredPercentage) /
+            _.keys(kv1.groupsSet).length;
+        const t2 =
+            _.sumBy(_.values(kv2.groupsSet), count => count.alteredPercentage) /
+            _.keys(kv2.groupsSet).length;
+        return t2 - t1;
```

Comment:
what is `kv1, kv2` and `t1 t2`? Could we give them a more readable name?

Reply:
Sure have named them.